src/lib/tools/planning/TodoManagerTool.ts

```
@@ -79,6 +79,24 @@ export function createTodoManagerTool(executionContext: ExecutionContext): Dynam
             todoStore.replaceAll(args.todos.map(t => t.content))
             resultMessage = `Replaced all TODOs with ${args.todos.length} new items`
             break
+          
+          case 'get_next':
+            const nextTodo = todoStore.getNextTodo()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)